### ISSUE TITLE
Big files

### ISSUE THREAD

**Issue by JamesDMSpeed** [anders-kolstad/sdmShiny#1]
For large files that we don't edit (raster datasets etc) we can try storing these in e.g. box and getting a direct download link for these. There is an example of this in https://github.com/JamesDMSpeed/PhyloTroph/blob/02c1f869fc6524e03b1789989275b1515079a5e7/R/Birds.R

**Comment by anders-kolstad** [comment]
Good idea. The combined raster stack with all environmental data is just under 100mb though.
